app/controllers/v2/application_controller.rb

```diff
@@ -60,5 +60,17 @@ def rbac_allowed?
       permission = self.class.instance_variable_get(:@action_permissions)[action_name.to_sym]
       user.authorized_to?(Rbac::INVENTORY_HOSTS_READ) && user.authorized_to?(permission)
     end
+
+    # Reduce through all the parents of the resource and join+scope them on the resource
+    def expand_resource
+      permitted_params[:parents].to_a.reduce(resource) do |scope, parent|
```

Comment:
Can this check for existence of `:parents` first? Or it isn't needed?

Reply:
`nil.to_a` returns `[]` empty array. Can you please add a comment for this case, that it would return `resource` "non-reduced"?